From 4008740d5ea3395c73c201dbc58fd0910031a392 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Armin=20Krezovi=C4=87?= <krezovic.armin@gmail.com>
Date: Fri, 30 Sep 2016 14:11:12 +0200
Subject: [PATCH 23/61] weston: Rename weston_output_init_pending() to
 weston_output_init()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

v2:

 - Rebased for latest changes.

v3:

 - Rebased for changes in wayland backend.

Reviewed-by: Quentin Glidic <sardemff7+git@sardemff7.net>
Reviewed-by: Pekka Paalanen <pekka.paalanen@collabora.co.uk>
Signed-off-by: Armin KrezoviÄ‡ <krezovic.armin@gmail.com>
---
 libweston/compositor-drm.c      | 2 +-
 libweston/compositor-fbdev.c    | 2 +-
 libweston/compositor-headless.c | 2 +-
 libweston/compositor-rdp.c      | 2 +-
 libweston/compositor-wayland.c  | 6 +++---
 libweston/compositor-x11.c      | 2 +-
 libweston/compositor.c          | 6 +++---
 libweston/compositor.h          | 8 +++-----
 8 files changed, 14 insertions(+), 16 deletions(-)

diff --git a/libweston/compositor-drm.c b/libweston/compositor-drm.c
index 491f171..567240f 100644
--- a/libweston/compositor-drm.c
+++ b/libweston/compositor-drm.c
@@ -2563,7 +2563,7 @@ create_output_for_connector(struct drm_backend *b,
 	b->crtc_allocator |= (1 << output->crtc_id);
 	b->connector_allocator |= (1 << output->connector_id);
 
-	weston_output_init_pending(&output->base, b->compositor);
+	weston_output_init(&output->base, b->compositor);
 	weston_compositor_add_pending_output(&output->base, b->compositor);
 
 	return 0;
diff --git a/libweston/compositor-fbdev.c b/libweston/compositor-fbdev.c
index 8b55332..0c45e98 100644
--- a/libweston/compositor-fbdev.c
+++ b/libweston/compositor-fbdev.c
@@ -498,7 +498,7 @@ fbdev_output_create(struct fbdev_backend *backend,
 	output->base.disable = NULL;
 	output->base.enable = fbdev_output_enable;
 
-	weston_output_init_pending(&output->base, backend->compositor);
+	weston_output_init(&output->base, backend->compositor);
 
 	/* only one static mode in list */
 	output->mode.flags =
diff --git a/libweston/compositor-headless.c b/libweston/compositor-headless.c
index ea23717..e7fc397 100644
--- a/libweston/compositor-headless.c
+++ b/libweston/compositor-headless.c
@@ -239,7 +239,7 @@ headless_output_create(struct weston_compositor *compositor,
 	output->base.disable = headless_output_disable;
 	output->base.enable = headless_output_enable;
 
-	weston_output_init_pending(&output->base, compositor);
+	weston_output_init(&output->base, compositor);
 	weston_compositor_add_pending_output(&output->base, compositor);
 
 	return 0;
diff --git a/libweston/compositor-rdp.c b/libweston/compositor-rdp.c
index b34024a..d385a31 100644
--- a/libweston/compositor-rdp.c
+++ b/libweston/compositor-rdp.c
@@ -573,7 +573,7 @@ rdp_backend_create_output(struct weston_compositor *compositor)
 	output->base.disable = rdp_output_disable;
 	output->base.enable = rdp_output_enable;
 
-	weston_output_init_pending(&output->base, compositor);
+	weston_output_init(&output->base, compositor);
 	weston_compositor_add_pending_output(&output->base, compositor);
 
 	return 0;
diff --git a/libweston/compositor-wayland.c b/libweston/compositor-wayland.c
index da0c4fe..580c7b5 100644
--- a/libweston/compositor-wayland.c
+++ b/libweston/compositor-wayland.c
@@ -1155,7 +1155,7 @@ wayland_output_create(struct weston_compositor *compositor, const char *name)
 
 	output->base.name = strdup(name);
 
-	weston_output_init_pending(&output->base, compositor);
+	weston_output_init(&output->base, compositor);
 	weston_compositor_add_pending_output(&output->base, compositor);
 
 	return 0;
@@ -1242,7 +1242,7 @@ wayland_output_create_for_parent_output(struct wayland_backend *b,
 		goto out;
 	}
 
-	weston_output_init_pending(&output->base, b->compositor);
+	weston_output_init(&output->base, b->compositor);
 
 	output->base.scale = 1;
 	output->base.transform = WL_OUTPUT_TRANSFORM_NORMAL;
@@ -1280,7 +1280,7 @@ wayland_output_create_fullscreen(struct wayland_backend *b)
 
 	output->base.name = strdup("wayland-fullscreen");
 
-	weston_output_init_pending(&output->base, b->compositor);
+	weston_output_init(&output->base, b->compositor);
 
 	output->base.scale = 1;
 	output->base.transform = WL_OUTPUT_TRANSFORM_NORMAL;
diff --git a/libweston/compositor-x11.c b/libweston/compositor-x11.c
index 86241c6..dadcd10 100644
--- a/libweston/compositor-x11.c
+++ b/libweston/compositor-x11.c
@@ -1031,7 +1031,7 @@ x11_output_create(struct weston_compositor *compositor,
 	output->base.disable = x11_output_disable;
 	output->base.enable = x11_output_enable;
 
-	weston_output_init_pending(&output->base, compositor);
+	weston_output_init(&output->base, compositor);
 	weston_compositor_add_pending_output(&output->base, compositor);
 
 	return 0;
diff --git a/libweston/compositor.c b/libweston/compositor.c
index 30eab1d..2f83313 100644
--- a/libweston/compositor.c
+++ b/libweston/compositor.c
@@ -4427,8 +4427,8 @@ weston_output_set_transform(struct weston_output *output,
  * configured either by compositors or backends.
  */
 WL_EXPORT void
-weston_output_init_pending(struct weston_output *output,
-			   struct weston_compositor *compositor)
+weston_output_init(struct weston_output *output,
+		   struct weston_compositor *compositor)
 {
 	output->compositor = compositor;
 	output->destroying = 0;
@@ -4587,7 +4587,7 @@ weston_output_enable(struct weston_output *output)
  *
  * \param output The weston_output object that needs to be disabled.
  *
- * See weston_output_init_pending() for more information on the
+ * See weston_output_init() for more information on the
  * state output is returned to.
  *
  * Calls a backend specific function to disable an output, in case
diff --git a/libweston/compositor.h b/libweston/compositor.h
index d3d779f..6cca4f3 100644
--- a/libweston/compositor.h
+++ b/libweston/compositor.h
@@ -1611,9 +1611,7 @@ void
 weston_output_update_matrix(struct weston_output *output);
 void
 weston_output_move(struct weston_output *output, int x, int y);
-void
-weston_output_init(struct weston_output *output, struct weston_compositor *c,
-		   int x, int y, int width, int height, uint32_t transform, int32_t scale);
+
 void
 weston_compositor_add_output(struct weston_compositor *compositor,
                              struct weston_output *output);
@@ -1830,8 +1828,8 @@ weston_output_set_transform(struct weston_output *output,
 			    uint32_t transform);
 
 void
-weston_output_init_pending(struct weston_output *output,
-			   struct weston_compositor *compositor);
+weston_output_init(struct weston_output *output,
+		   struct weston_compositor *compositor);
 
 void
 weston_compositor_add_pending_output(struct weston_output *output,
-- 
2.7.4

